.changeset/platform-workspace-exec-transport-logs.md

```
@@ -0,0 +1,9 @@
+---
+'@mastra/platform-workspace': patch
+---
+
+Add warnings that make it easier to diagnose slow sandbox commands.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)